src/utils/use-in-app-review.ts

```diff
@@ -0,0 +1,57 @@
+import InAppReview from 'react-native-in-app-review';
+import {useAnalyticsContext} from '@atb/analytics';
+import {useFeatureTogglesContext} from '@atb/feature-toggles';
+import {useCallback} from 'react';
+import {notifyBugsnag} from '@atb/utils/bugsnag-utils';
+
+export function useInAppReviewFlow() {
+  const analytics = useAnalyticsContext();
+  const {isInAppReviewEnabled} = useFeatureTogglesContext();
+
+  const requestReview = useCallback(async () => {
```

Comment:
If we want the `requestReview` to just be fire and forget, it should return `void` instead of `Promise<void>`, so IDEs doesn't complain about unhandled promise.